source/tutorials/kubernetes.rst

```diff
@@ -3,354 +3,657 @@
 Kubernetes\*
 ############
 
-This tutorial describes how to install, configure, and start the
-`Kubernetes container orchestration system`_ on |CL-ATTR| using CRI+O and kata-runtime.
+This tutorial describes how to install, configure, and start the `Kubernetes
+container orchestration system`_ on |CL-ATTR|.
+
+A Kubernetes cluster can be setup on |CL| using scripts to automated the
+process or manually step-by-step.
```

Comment:
automated should be automate.  I would rephrase to say "A kubernetes cluster can be setup on CL using the CL cloud-native-setup scripts to automate the process or can be setup through a manual step by step process.  This tutorial covers both scenarios."  I would have in page hyperlinks to the section for the scripts and the section for manual process.  I don't just like saying "scripts" without any context on where they came from so I think adding in they are Clear Linux cloud-native-setup scripts gives more confidence this isn't just some unmaintained script somewhere.